actions/terra-k8s-linter/policy/deployments/deny.rego

```diff
@@ -0,0 +1,32 @@
+package main
+
+import data.kubernetes
+
+
+name = input.metadata.name
+
+required_deployment_labels {
+  input.metadata.labels["app.kubernetes.io/name"]
+  input.metadata.labels["app.kubernetes.io/instance"]
+  input.metadata.labels["helm.sh/chart"]
+  input.metadata.labels["app.kubernetes.io/managed-by"]
+}
+
+deny_required_deployment_labels[msg] {
+  input.kind == "Deployment"
+  not required_deployment_labels
+  msg = sprintf("%s must include Kubernetes recommended labels", [name])
+}
+
+deny_replicas[msg] {
+  input.kind == "Deployment"
+  input.metadata.annotations["bio.terra.linter/replicas_exception"] != "enabled"
+  input.spec.replicas < 3
```

Comment:
Probably something to bring up in norms, but I'm generally a little leery about having unaccompanied numbers in code, would prefer to have some sort of static like MIN_REQUIRED_REPLICAS that we can set and then reference it here so there's a little more context as to what this number means and if we need to change it in the future, why is was set (e.g. if it was arbitrary or some other reason)

Reply:
This is defined in the README. Each action is required to have it. The static variable name is a great suggestion
33cc7c9